src/terra-dev-site/about/ComponentStandards.b.about.mdx

```diff
@@ -109,6 +99,25 @@ Components should be internationalized and support the following locales:
 * Swedish `sv`
 * Swedish (Sweden) `sv-SE`
 
-### LTR / RTL Support
+## LTR / RTL
+
+Terra support both LTR and RTL directionalities. This is largely handled by our use of [postcss-rtl](https://github.com/vkalinichev/postcss-rtl). When working on CSS changes, be sure to test in RTL.
+
+## Themeing
+
+Terra components support the following themes:
+
```

Comment:
Should we mention the default theme as well?

Reply:
Yeah probably

Reply:
what do we call it?

Reply:
Good question

Reply:
We could avoid the question:

```
Terra components support the following additional themes:

* clinical-lowlight-theme
* orion-fusion-theme
```